lib/main.dart

```diff
@@ -193,7 +193,9 @@ class _InsporationState extends State<Insporation> {
         _navigator.currentState.pushNamed("/publisher", arguments: PublisherOptions(prefill: prefill));
         break;
       case "images":
+        final text = shareEvent["text"]?.isNotEmpty == true ? shareEvent["text"] : "";
```

Comment:
```suggestion
        final text = shareEvent["text"]?.isNotEmpty == true ? shareEvent["text"] : null;
```

Reply:
Sure? prefill defaults to "" in method parameter. ( `this.prefill = ""` )

Reply:
Right, nvm, I was looking at line 190 above for comparison.